src/pages/main/components/carousel/Carousel.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import CarouselImg from './CarouselImg';
+import './Carousel.scss';
+
+const Carousel = () => {
+  const [imgList, setImgList] = useState([]);
+  const [curImg, setCurImg] = useState(0);
+  const [fadeIn, setFadeIn] = useState(true);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/CAROUSEL_DATA.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setImgList(data);
+      });
+  }, []);
+
+  const moveToImg = targetNum => {
+    setFadeIn(prev => !prev);
+    setTimeout(() => {
+      setFadeIn(prev => !prev);
+      setCurImg(targetNum);
+    }, 400);
+  };
+
+  return (
+    <section className="carousel">
+      <div className="carouselWrapper">
+        {imgList.map(imgData => {
+          return (
+            <CarouselImg
+              key={imgData.id}
+              imgData={imgData}
+              curImg={curImg}
+              fadeIn={fadeIn}
+            />
+          );
+        })}
```

Comment:
```suggestion
      {imgList.map((imgData) => (
        <CarouselImg
          key={imgData.id}
          imgData={imgData}
          curImg={curImg}
          fadeIn={fadeIn}
        />
      ))}
```
소괄호를 사용하면 return 을 생략해서 사용할 수도 있습니다!